src/plugins/kibana_legacy/public/font_awesome/font_awesome.scss

```diff
@@ -0,0 +1,32 @@
+@font-face {
```

Comment:
Can you move this whole FA folder to https://github.com/elastic/kibana/tree/master/src/legacy/ui/public/styles so we can manage it alongside all the other legacy styles?

Reply:
The legacy platform will get removed soon, but we won't be able to completely migrate away from font awesome till then. `kibana_legacy` is the right home for things like that which are scheduled to be removed from Kibana soon, but have to work in the new platform for a short while. I already created an issue for other similar leftovers in `ui/styles`: https://github.com/elastic/kibana/issues/69135

Reply:
We talked this out on Slack too, but apparently this method implemented for FA needs to happen to all the styles in `ui/styles` and the App team will take care of that.